Update pull_request_template.md (#21)
DataDog/datadog-lambda-layer-rb: .github/pull_request_template.md

> <!--- Please remember to review the [contribution guidelines](https://github.com/DataDog/datadog-lambda-layer-python/blob/master/CONTRIBUTING.md) if you have not yet done so._  --->

Reviewer: Should we copy the Python contribution guidelines here and link to them in the Ruby repo?

Author: That's a good catch.

### What does this PR do?

<!--- A brief description of the change being made with this pull request. --->

### Motivation

<!--- What inspired you to submit this pull request? --->

### Testing Guidelines

<!--- How did you test this pull request? --->

### Additional Notes

<!--- Anything else we should know when reviewing? --->

### Types of changes

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change
- [ ] Misc (docs, refactoring, dependency upgrade, etc.)

### Checklist

> - [ ] This PR's description is comprehensive
> - [ ] This PR contains breaking changes that are documented in the description
> - [ ] This PR introduces new APIs or parameters that are documented and unlikely to change in the foreseeable future
> - [ ] This PR impacts documentation, and it has been updated (or a ticket has been logged)
> - [ ] This PR's changes are covered by the automated tests
> - [ ] This PR collects user input/sensitive content into Datadog

Reviewer: ```suggestion
- [ ] This PR's description is comprehensive
- [ ] Any breaking changes contained in this PR are documented in the description
- [ ] Any new APIs or parameters introduced in this PR are documented and unlikely to change in the foreseeable future
- [ ] Any necessary documentation changes have been made (or a ticket generated)
- [ ] This PR's changes are covered by the automated tests
- [ ] This PR collects no user input/sensitive content into Datadog
```

Reviewer: Nitpick: I tried to modify this checklist to affirm things we want.

Author: Hmm, this is not what I meant. I think the checklist serves as a remainder, but you only check the ones applicable to your PR.

Reviewer: Okay. It's probably good enough as is. Maybe we might want to rename the section "Check all that apply"? Checklist makes me think of a list of things I need to do before submitting the PR.

Author: Yes, I will add a sentence "Check all that apply".

Author: I'm actually trying to avoid people checking all the boxes for every pr, based on experience, after a while, people will start checking all the boxes without reading or thinking about them.  When you don't have to check all the boxes, it kinda forces you to read through the list every pr.
